enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionDefaultView/InteractiveQuestionDefaultView.tsx

```diff
@@ -114,9 +113,7 @@ export const InteractiveQuestionDefaultView = ({
   }
 
   const showSaveButton =
-    shouldShowSaveButton({ question, originalQuestion }) &&
-    isSaveEnabled &&
-    !isSaveModalOpen;
+    shouldShowSaveButton({ question, originalQuestion }) && isSaveEnabled;
```

Comment:
If we hide the save question when showing the modal, the `New question` header will jump as the save button is taller than the header. Also, [Poom confirmed](https://metaboat.slack.com/archives/C063Q3F1HPF/p1752050888981989?thread_ts=1752048433.170339&cid=C063Q3F1HPF) we can remove this condition.